### PR TITLE
Update embedded logos

### PR DESCRIPTION
**Pull request by carkod**
## Done

Update logos

## QA

- Check https://ubuntu-com-12542.demos.haus/embedded has the [new logos](https://warthogs.atlassian.net/browse/WD-1271?atlOrigin=eyJpIjoiNjhjMDY3MTdhYzA0NDg3OWE1YzExZWFjYWVhOTRiZTciLCJwIjoiaiJ9)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1271?atlOrigin=eyJpIjoiZDMwNTlkZDM5ZGE1NDYzZjk4MWMwMTUyNjhiOWZjMTMiLCJwIjoiaiJ9

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by camillet16** [comment]
looks good everything has a prefix! woohoo. One thing I checked the code, if you could please add the alts i.e. alt='Canonical LXD' for the section with docker, LXD, Snaps, Kubernetes logos? There are no text titles there so it would not be accessible for screen readers. Future thanks.

**Comment by camillet16** [comment]
Another note the upper left corner of both Canonical Snaps and Canonical LXD orange tag has a rounded border.

**Comment by carkod** [comment]
> Another note the upper left corner of both Canonical Snaps and Canonical LXD orange tag has a rounded border.

That was weird, thanks! For some reason images have rounded corders by default in this project.

**Comment by camillet16** [comment]
💯  Thank you they look great!

File: templates/embedded/index.html [hunk 1]
```diff
@@ -494,34 +494,31 @@ <h3>All kinds of containers</h3>
...
           loading="lazy",
-          attrs={"class": "p-card__thumbnail", "style": "max-height: 4rem;"}
+          attrs={"class": "p-card__thumbnail", "style": "border-radius: 0"}
         ) | safe
       }}
       <hr class="u-sv1">
-      <h3>Snaps</h3>
       <p>App packages for desktop, cloud and IoT that are easy to install, secure, cross‐platform and dependency‐free. Snaps are discoverable and installable from the Snap Store, the app store for Linux with an audience of millions.</p>
-      <p><a href="https://snapcraft.io/first-snap">Build a snap</a></p>
+      <p class="p-card__content"><a href="https://snapcraft.io/first-snap">Build a snap</a></p>
```
Comment (pedoch) [inline templates/embedded/index.html:508]: Is there any need for the `p-card__content class`? The links in other cards don't have it. I don't know if there's any downside per se but I was just wondering. I'm going to approve anyway, everything looks good.

**Review by pedoch — APPROVED** [review]
LGTM 👍

File: templates/embedded/index.html (hunk 1 above)
Comment (carkod) [reply templates/embedded/index.html:508]: It should be wrapped around the content maybe. It reduces a bit the spacing, but in this case I don't think it makes a big difference. I can remove it

**Pull request merged**